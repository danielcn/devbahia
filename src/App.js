import React from 'react';
import Button from '@material-ui/core/Button';
import Card from '@material-ui/core/Card';
import CardActions from '@material-ui/core/CardActions';
import CardContent from '@material-ui/core/CardContent';

const App = () =>(
    <Card className="App">
        <CardContent>
            <h1>Hello Dev Bahia</h1>
            <CardActions>
                <Button size="small" color="primary">Learn More</Button>
            </CardActions>
        </CardContent>
    </Card>
);

export default App;
