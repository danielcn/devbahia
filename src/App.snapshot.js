import React from 'react';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

import App from './App';

describe('App Snapshot', () => {
    test('renders', () => {
      const component = renderer.create(
        <App />
      );
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

    test('should render without throwing an error', function() {
      const wrapper = shallow(<App />);
      expect(wrapper.contains(<h1>Hello Dev Bahia</h1>)).toBe(true);
    });

});
