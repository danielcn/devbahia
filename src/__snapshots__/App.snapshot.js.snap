// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot renders 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1 App"
>
  <div
    className="MuiCardContent-root-29"
  >
    <h1>
      Hello Dev Bahia
    </h1>
    <div
      className="MuiCardActions-root-30"
    >
      <button
        className="MuiButtonBase-root-58 MuiButton-root-32 MuiButton-text-34 MuiButton-textPrimary-35 MuiButton-flat-37 MuiButton-flatPrimary-38 MuiButton-sizeSmall-55 MuiCardActions-action-31"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiButton-label-33"
        >
          Learn More
        </span>
        <span
          className="MuiTouchRipple-root-61"
        />
      </button>
    </div>
  </div>
</div>
`;
